src/components/Thread/Thread.vue

```diff
@@ -674,10 +674,11 @@ export default {
         text_color (message) {
             try {
                 let colorString;
-                if (message.message_from)
-                    colorString = this.getColor(message);
-                else // Otherwise default color
+
+                if (message.type == 1) // Message was sent by the user, use default color
                     colorString = this.color;
+                else
+                    colorString = this.getColor(message);
```

Comment:
This actually isn't really accurate. Messages sent by the user can have a number of [different types](https://github.com/klinker-apps/pulse-sms-android/blob/master/shared/src/main/java/xyz/klinker/messenger/shared/data/model/Message.kt#L147):

* Sent (1)
* Sending (2)
* Failed (3)
* Delivered (4)

In this case, it doesn't really matter. When it gets down in to the `getColor` function, any messages without a `message_from` property won't go anywhere. Is there a reason that you made this change? I wouldn't have actually changed anything at all and I would rather have it be the way it used to be, since that is how the `getColor` function actually works.

This all should really just be simplified to 

```javascript
const colorString = this.getColor(message)
```

Reply:
@klinker24 During debugging, it seemed like the `message_from` was always null for any chat which was not a group chat. This caused every message to compare to the default color. I mistakenly thought the default color was the color of the messages sent by the user.

I can revert this and create a new PR if you'd like.

Reply:
Don't worry about it. I will make the changes